### PR TITLE
Brazilian portuguese

### PR DESCRIPTION
**Pull request by FKrauss**
as per the documentation I have "integrated" the pt-br language into the website and tested it locally.

Images are all in place, text is good and a few corrections were made in directly in the code so it wouldn't overflow their space.

I made a couple additions to it too:
1.  I started a small tutorial on how to integrate a new language - it's not 100% great but it is a start. I'll add more to it when I integrate another language
2. included a small telegram logo in svg in the `images/community` folder
3. included said telegram logo in the pt-BR page for community with the link to the Brazilian telegram support group
4. removed a bunch of whitespace in the `dao_content_tr.yml` file

Let me know what you think!

Cheers!

### CODE REVIEW

File: pt-BR/community.md
Comment (m52go) [inline pt-BR/community.md]:
```suggestion
    <li><img src="/images/community/youtube.svg" alt="youtube"> <a href="https://www.youtube.com/c/bisq-network">Canal Bisq no YouTube</a><br>Assista e participe de nossos calls semanais sobre crescimento. Veja entrevistas com desenvolvedores e outros conteúdos em vídeo</li>
```

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
  runningWindows: Parece que você está usando um Windows
```

With the period here, there's a double-period.

Screenshot from 2019-11-10 18-07-31

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
  runningMac: Parece que você está usando um Mac
```

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
  runnigDebian: Você parece estar usando um Debian/Ubuntu
```

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
  runningRedHat: Parece que você está usando um Red Hat/Fedora
```

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
  runningWindows: Parece que você está usando um Windows
```

For consistency, as none of the other languages include a period here. This variable and the other one should be merged since they're exactly the same, but that's for another time.

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
  runningMac: Parece que você está usando um Mac
```

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
  runnigDebian: Parece que você está usando um Debian/Ubuntu
```

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
  runningRedHat: Parece que você está usando um Red Hat/Fedora
```

File: pt-BR/faq.html
Comment (m52go) [inline pt-BR/faq.html]:
```suggestion
outdated_translation: false
```

Pages marked as outdated show this message at the top of the page.

Screenshot from 2019-11-10 18-12-02

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
  pSoftwareDownloadsVersion: Para v1.1.7 (aproximadamente)
```

File: _data/dao_content_tr.yml
Comment (m52go) [inline _data/dao_content_tr.yml]:
```suggestion
  img: /pt-BR/images/DAO/dao_intro.svg
```

File: _data/dao_content_tr.yml
Comment (m52go) [inline _data/dao_content_tr.yml]:
```suggestion
  img: /pt-BR/images/DAO/dao_why.svg
```

File: _data/dao_content_tr.yml
Comment (m52go) [inline _data/dao_content_tr.yml]:
```suggestion
  img: /pt-BR/images/DAO/dao_what.svg
```

File: _data/dao_content_tr.yml
Comment (m52go) [inline _data/dao_content_tr.yml]:
```suggestion
  img: /pt-BR/images/DAO/dao_how.svg
```

File: _data/dao_content_tr.yml
Comment (m52go) [inline _data/dao_content_tr.yml]:
```suggestion
  img: /pt-BR/images/DAO/dao_benefits.svg
```

**Review by m52go — CHANGES_REQUESTED** [review]
@FKrauss -- some notes:

This PR was a bit hard to review as-is because of the commit history...if you look at the "Files Changed" tab on the top of this page, you'll see this PR includes a lot of changes from the extra unrelated commits. I ended up cherry-picking the commits you made to do a proper review, and would need you to do the same to clean this PR up before merging. Ideally it would also be rebased onto a more recent version of the master branch.

I did it already [here on my repository](https://github.com/m52go/bisq-website/commits/pt-br-2). I can explain how to do it if you like, or you can give me push access to your bisq-website repository and I can just push what I have.

Some corrections:

1. ~These links on the bottom of the DAO page are missing spaces.~ I've added a suggested commit to fix this issue.

2. ~All pictures on the DAO page are still in English. Would you mind translating them? They're SVGs so you can manipulate the text in the SVG file.~ I've suggested corrections that fix this issue.

3. It looks like only some FAQ updates from the v1.2 PR made it here. For example, the payment methods section isn't current on this PR. Please check out [this PR](https://github.com/bisq-network/bisq-website/pull/272) for reference on everything that was updated.

4. I think it would be better to make the `language` attribute in page front matter `language: Português` instead of `language: Português (BR)`. The flag accomplishes the distinction between PT-PT and PT-BR.

I think this is everything, but it will be easier to be more confident once the cruft is removed and these corrections are made.

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
    <p>Veja mais recursos em nosso <a href='https://docs.bisq.network/dao.html'>Tudo Sobre a DAO Bisq</a> doc.</p><p>Contribuidores anteriores, verifiquem <a href='https://docs.bisq.network/compensation.html#get-added-to-genesis'>o link</a> para obter instruções sobre como ser adicionado à distribuição de BSQ da gênese.</p>
```

File: _data/pt-BR.yml
Comment (FKrauss) [reply _data/pt-BR.yml]: I changed these in Transifex now, so they won't be a prob moving forward

File: _data/pt-BR.yml
Comment (FKrauss) [reply _data/pt-BR.yml]: also fixed in Transifex

File: _data/dao_content_tr.yml
Comment (FKrauss) [reply _data/dao_content_tr.yml]: Ok, for the SVG images I need to look into it first

File: _data/pt-BR.yml
Comment (m52go) [inline _data/pt-BR.yml]:
```suggestion
    <p>Bisq está sempre à procura de desenvolvedores, designers e especialistas em segurança altamente qualificados para se juntarem à nossa <a href="https://bisq.network/community/">comunidade</a>. Se você tem interesse em  melhorar a Bisq, há muitas maneiras de <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">contribuir</a>. Gostaríamos muito de ouvir você!</p> <p>O nosso <a href="https://bisq.network/roadmap/">roadmap</a> oferece uma visão geral dos recursos e marcos de alto nível no caminho para a Bisq v1.0.</p>
```

Removed extra link to community page, to keep consistent with changes from #287.

**Comment by m52go** [comment]
@FKrauss I've integrated all the feedback except the FAQs, which need to be updated in Portuguese (item 3 from my [review comment](https://github.com/bisq-network/bisq-website/pull/281#pullrequestreview-314647748)).

**Pull request closed without merging**
